File: c/media_pares.c
#include <stdio.h>

int main(){
    int n, somapares = 0, npares = 0;
    double mediapares;

    printf("Quantos elementos vai ter o vetor? ");
    scanf("%d", &n);

    int vetor[n];

    for (int i=0; i<n; i++) {
        printf("Digite um numero: ");
        scanf("%d", &vetor[i]);
    }

    for (int i=0; i<n; i++) {
        if (vetor[i] % 2 == 0) {
            somapares = somapares + vetor[i];
            npares++;
        }
    }

    if (npares == 0) {
        printf("NENHUM NUMERO PAR\n");
    }
    else {
        mediapares = (double)somapares / npares;

        printf("MEDIA DOS PARES = %.1lf\n", mediapares);
    }
	
    return 0;
}

File: python/media_pares.py
n: int; somapares: int; npares: int
mediapares: float

n = int(input("Quantos elementos vai ter o vetor? "))

vetor: [int] = [0 for x in range(n)]

for i in range(n):
	vetor[i] = int(input("Digite um numero: "))

npares = 0
somapares = 0
for i in range(n):
	if vetor[i] % 2 == 0:
		somapares = somapares + vetor[i]
		npares = npares + 1

if npares == 0:
	print("NENHUM NUMERO PAR")
else:
	mediapares = float(somapares) / npares

	print(f"MEDIA DOS PARES = {mediapares:.1f}")